components/ui/AboutContent.jsx

```diff
@@ -0,0 +1,64 @@
+import data from "../../data/aboutContent";
```

Comment:
Reuse the component in the `About.jsx` for tablet and desktop instead of having repeatition